orga/tests/test-helper.js

```diff
@@ -1,7 +1,7 @@
 import Application from '../app';
 import config from '../config/environment';
 import { setApplication } from '@ember/test-helpers';
-import { start } from 'ember-qunit';
+import start from 'ember-exam/test-support/start';
```

Comment:
💡, pour ceux qui ne le savent pas : 
[Ember exam](https://github.com/ember-cli/ember-exam) permet de contrôler la manière de lancer les tests (split / parallélisation ...). Pour l'utiliser il fallait remplacer le `start` venant de qunit par le `start` d'`ember-exam` ([voir ici](https://github.com/ember-cli/ember-exam#how-to-use))